tools/scripts/update-docker-tags.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+# Retags all docker images from latest Environments build
+# bash tools/scripts/update-docker-tags.sh OLD_VERSION NEW_VERSION
+
+# check for dirty changes
+if [[ -n "$(git status --porcelain)" ]]; then
+    echo "untracked or dirty files are not allowed, cleanup before running update-docker-tags.sh"
+    exit 1
+fi
+
+# parse version args
+export OLD_VERSION="$1"
+export NEW_VERSION="$2"
+
+# get list of tags to replace via OLD_VERSION in bumpenvs.yaml
+export IMAGES=$(grep -o -P "(?<=new: ).*(?=,)" tools/scripts/bumpenvs.yaml | grep $OLD_VERSION)
```

Comment:
This will be changed to read from a separate file that tracks currently published images. That file will live in the environments repo: https://github.com/determined-ai/environments